### PR TITLE
Fix zero length interval issue in Junit emitter

### PR DESCRIPTION
**Pull request by itaysabato**

### CODE REVIEW

File: src/midje/emission/plugins/junit.clj [hunk 1]
```diff
@@ -64,9 +64,12 @@
       (str (fact/file fact) ":" (fact/line fact))))
 
 (defn process-fact [fact]
-  (let [elapsed (/ (time/in-millis (time/interval (-> fact :attrs :start-time)
-                                                  (-> fact :attrs :stop-time)))
+  (let [start-time (-> fact :attrs :start-time)
+        stop-time (-> fact :attrs :stop-time)
+        elapsed (if (time/before? start-time stop-time) 
+                  (/ (time/in-millis (time/interval start-time stop-time))
                    1000.0)
```
Comment (philomates) [inline src/midje/emission/plugins/junit.clj:71]: unfortunately there is no auto-linting on the project so we need to do it manually
here the `1000.0` needs one more level of indentation to match up with the first arg of the `/`

**Review by philomates — COMMENTED** [review]
thanks for contributing! I can merge and deploy this after you address the one whitespace comment

File: src/midje/emission/plugins/junit.clj (hunk 1 above)
Comment (itaysabato) [reply src/midje/emission/plugins/junit.clj:71]: Right! On it.

Comment (itaysabato) [reply src/midje/emission/plugins/junit.clj]: Done.

**Comment by philomates** [comment]
deployed as `1.9.10`

**Pull request merged**